sdk/eventhub/Azure.Messaging.EventHubs/src/Resources.resx

```diff
@@ -171,4 +171,7 @@
   <data name="ResourceMustMatchSharedAccessSignature" xml:space="preserve">
     <value>The reqested resource, '{0}', does not match the resource of the shared access signature, '{1}'.   A token cannot be issued.</value>
   </data>
+  <data name="InvalidConnectionType" xml:space="preserve">
```

Comment:
Should this resource now be `InvalidTransportType`? along with an updated message `The requested transport type, '{0}' is not supported.`?